discovery_engine/lib/src/ffi/types/engine.dart

```diff
@@ -227,6 +229,13 @@ class DiscoveryEngineFfi implements Engine {
         .toDocumentListWithActiveData(isSearched: true);
   }
 
+  /// Returns the currently trending topics.
+  @override
+  Future<List<TrendingTopic>> getTrendingTopics() async {
+    // TODO: implement engine call
+    throw UnimplementedError();
```

Comment:
:art: the message could be added directly to the error, maybe with the corresponding ticket number
```suggestion
    throw UnimplementedError('TODO: implement engine call');
```